spec/helper.rb

```diff
@@ -20,7 +20,7 @@
   host: "127.0.0.1",
   database: "delayed_job_test",
   username: "root",
-  password: "root",
+  password: "",
```

Comment:
This was needed to get the tests passing locally. Not sure if it's something we'd want to merge or if it's just something I needed!

Reply:
It's probably because you're running a local MySQL instance without a password - perhaps for now we can `ENV.fetch("MYSQL_PASSWORD", "")` ?

Reply:
(or default to root, which is what is used in the docker-compose config)